### DISCUSSION TITLE
Make finding rarely cooked recipies easier

### DISCUSSION THREAD

**Discussion by sibbl** [mealie-recipes/mealie#1919] [Feature Request]
**Before submitting this feature request I have**
- [x] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x] Reviewed the existing feature requests and my feature has not been requested.
- [x] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
Some recipes are cooked more often, some less often, when filling the meal plan manually (or even randomly).
It would be awesome to find those recipies which haven't been on the meal plan for a long time.

**(Optional): Suggest A Solution**
* In the recipe detail view (besides the "comments") there could be shown a log, including an entry each time it was on the meal plan.
* When randomly filling the meal plan, recipies which haven't been on the meal plan for a long time, could be preferred.
* When scrolling through recipies, sorting by the last time it was on the meal plan would be helpful.
* It could maybe make sense to allow sorting by "number of times recipies were on the meal plan in a given time span (1 month, 1 year, ...)"

Feedback and more ideas are very welcome!

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by michael-genson** [comment]
We recently implemented a "last made this" feature in nightly, along with a new recipe events timeline:
https://github.com/hay-kot/mealie/pull/1801
https://github.com/hay-kot/mealie/pull/1831

I'd like to get this integrated with meal plans, and potentially count the number of times a recipe is made, which would more-or-less achieve this :)

**Comment by sibbl** [comment]
Awesome, thanks for the reply and the links to the issues.

Seems like I searched for all keywords like "history" or "cooked" but not "made this" or "timeline"...

I've just set up the nightly and it seems indeed to match my requirements. Will fiddle around a bit more with this great piece of software. Thank you :)
